utils.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+
+from vectara_agentic.agent import Agent
+
+from query_vectara import VectaraQuery
+
+
+def query_vectara(query, convo_id, vectara_prompt, bot_type):
+    if query.startswith('!agentic'):
+        agent = Agent.from_corpus(
+            vectara_customer_id=os.getenv("VECTARA_CUSTOMER_ID"),
+            vectara_corpus_id=os.getenv("VECTARA_CORPUS_IDS"),
+            vectara_api_key=os.getenv("VECTARA_API_KEY"),
+            data_description="Vectara website, docs and forum data ",
```

Comment:
This needs to be a parameter. Again as above, in the configuration, if the person installing RAGTIME wants to use agentic they should be able to define in configuration the description and the specialty. Should not be fixed in code like this.